sr_description/mujoco_models/shared_assets.xml

```diff
@@ -9,6 +9,20 @@
     <material name="metal" reflectance="1.0" rgba="0.6 0.6 0.6 1.0" shininess="0.5" specular="1.0"/>
     <material name="plastic" rgba="0.2 0.2 0.2 1.0" shininess="0.5"/>
     <material name="rubber" rgba="0.3 0.3 0.3 1.0" shininess="0.0"/>
+    <mesh name="base" file="meshes/arm_and_hand_meshes/visual/base.stl" scale="1 1 1"/>
```

Comment:
This means the UR meshes will be loaded even when we are running e.g. only hand. They should probably be in a seperate, UR-specific assets .xml.

Reply:
Also, ideally UR meshes would be imported from ur_description, not duplicated here. I guess Mujoco xml syntax doesn't allow resolving ROS package paths though. Maybe we could have a setup script that creates symlinks? Possibly not worth it.